test/support/schema.sql

```diff
@@ -14,3 +14,15 @@ CREATE TABLE `clients` (
   `updated_at` datetime DEFAULT NULL,
   PRIMARY KEY (`id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci;
+
+DROP TABLE IF EXISTS `users`;
+CREATE TABLE `users` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT NULL,
+  `email` varchar(255) DEFAULT NULL,
+  `crypted_password` varchar(255) DEFAULT NULL,
+  `password_salt` varchar(255) DEFAULT NULL,
```

Comment:
Yeah, I wrote eboshi back in 2008, so the schema reflects that somewhat. I think this was before bcrypt hit the scene, and sha256 was still all the rage.